web/src/app/modules/sugarloaf/components/smart/namespace/namespace.component.html

```diff
@@ -1,10 +1,27 @@
 <div class="app-namespace-selector">
-  <span class="title">Namespace</span>
-  <ng-select
-    class="namespace-dropdown"
-    [items]="namespaces"
-    (change)="selectNamespace($event)"
-    [(ngModel)]="currentNamespace"
-    [clearable]="false"
-  ></ng-select>
+  <ng-template [ngIf]="showDropdown()" [ngIfElse]="hideDropdown">
+    <clr-dropdown [clrCloseMenuOnItemClick]="true">
+      <button type="button" clrDropdownTrigger>
+        <clr-icon shape="namespace"></clr-icon>
+      {{ currentNamespace }}
+        <clr-icon shape="caret down"></clr-icon>
+      </button>
+      <clr-dropdown-menu *clrIfOpen [clrPosition]="'bottom-right'">
+        <label class="dropdown-header">Namespaces</label>
+        <ng-container *ngFor="let namespace of namespaces;trackBy: trackByIdentity">
+          <button
+            type="button"
+            [ngClass]="namespaceClass(namespace)"
+            clrDropdownItem
+            (click)="selectNamespace(namespace)">
+          {{ namespace }}
+          </button>
+        </ng-container>
+      </clr-dropdown-menu>
+    </clr-dropdown>
+  </ng-template>
+  <ng-template #hideDropdown>
+    <clr-icon shape="namespace"></clr-icon>
+    [All Namespaces]
```

Comment:
Hit a case where navigation from `/cluster-overview` to `/` via top left octant logo results in stuck on `All Namespaces`

Reply:
Done